pubspec.yaml

```diff
@@ -16,6 +16,6 @@ homepage: https://github.com/QuiverDart/quiver_collection
 environment:
   sdk: '>=1.12.0 <2.0.0'
 dependencies:
-  quiver: '>=0.21.4 <0.22.0'
+  quiver_iterables: '>=1.0.0-dev.1 <1.0.0'
```

Comment:
Paranoid we might make breaking changes for 1.0.0. I suspect the chances are pretty amazingly low at this point, but given that we intend to publish them as 1.0.0 soon, shouldn't be an issue.